lib/rules/media-feature-name-value-whitelist/index.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+const _ = require("lodash");
+const atRuleParamIndex = require("../../utils/atRuleParamIndex");
+const matchesStringOrRegExp = require("../../utils/matchesStringOrRegExp");
+const mediaParser = require("postcss-media-query-parser").default;
+const report = require("../../utils/report");
+const ruleMessages = require("../../utils/ruleMessages");
+const validateOptions = require("../../utils/validateOptions");
+
+const ruleName = "media-feature-name-value-whitelist";
+
+const messages = ruleMessages(ruleName, {
```

Comment:
Formatters will usually append the rule name, therefore the "media feature" bit is a little redundant. Let's go with the following instead:

```
const messages = ruleMessages(ruleName, {
  rejected: (name, value) =>
    `Unexpected value "${value}" for name "${name}"`
});